pkg/plugin/v3/scaffolds/internal/templates/config/webhook/kustomization.go

```diff
@@ -37,13 +40,14 @@ func (f *Kustomization) SetTemplateDefaults() error {
 
 	f.TemplateBody = kustomizeWebhookTemplate
 
-	f.IfExistsAction = file.Error
+	// If file exists (ex. because a webhook was already created), skip creation.
+	f.IfExistsAction = file.Skip
 
 	return nil
 }
 
 const kustomizeWebhookTemplate = `resources:
-- manifests.yaml
+- manifests{{ if and (.WebhookVersion) (ne .WebhookVersion "v1") }}.{{ .WebhookVersion }}{{ end }}.yaml
```

Comment:
I think that you should probably set a default `WebhookVersion` in `SetTemplateDefaults` and simplify this conditional.